README.md

```diff
@@ -194,6 +194,11 @@ fd … -X ls -lhd --color=always
 This pattern is so useful that `fd` provides a shortcut. You can use the `-l`/`--list-details`
 option to execute `ls` in this way: `fd … -l`.
 
+Another example of the `-X` option can be shown with `rg`, using all the search results as arguments:
+```bash
+fd … -X rg -i target
```

Comment:
Maybe we could try to find a realistic example here to make it easier to understand what's going on? I would leave out the `-i` option for ripgrep. `target` also doesn't seem like a "typical" search pattern for me. Maybe something like:
```bash
fd -e cpp -e cxx -e h -e hpp -X rg 'std::cout'
```